include/librealsense2/utilities/concurrency/CMakeLists.txt

```diff
@@ -1,10 +1,13 @@
 # License: Apache 2.0. See LICENSE file in root directory.
 # Copyright(c) 2022 Intel Corporation. All Rights Reserved.
 
-set( CONCURRENCY_FILES
+set( CONCURRENCY_HEADER_FILES
     "${CMAKE_CURRENT_LIST_DIR}/concurrency.h"
+)
+set( CONCURRENCY_SOURCE_FILES
     "${CMAKE_CURRENT_LIST_DIR}/dispatcher.cpp"
-    )
+)
+set( CONCURRENCY_FILES ${CONCURRENCY_HEADER_FILES} )
```

Comment:
Oh it is here, so why do we also need it outside?

Reply:
I'll check

Reply:
Next PR it is...